.github/workflows/OCV-PR-4.x-U20-Cuda.yaml

```diff
@@ -0,0 +1,215 @@
+name: OCV PR:4.x U20 CUDA
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/OCV-PR-4.x-U20-Cuda.yaml'
```

Comment:
This workflow seems to test cuda modules from opencv_contrib. Should it be renamed to `.github/workflows/OCV-Contrib-PR-4.x-U20-Cuda.yaml`?

Reply:
I think it would be better to have CUDA pipeline for each repository. I've added one more workflow in the latest commit.